main.go

```diff
@@ -149,6 +159,9 @@ func visit(url *url.URL) {
 	if err != nil {
 		log.Fatalf("unable to create request: %v", err)
 	}
+	for _, h := range httpHeaders {
+		req.Header.Add(headerKeyValue(h))
```

Comment:
I think this is ok, the host comes from the URL, see #36 for the `-x` flag that lets you send the request to a different iP.